test/integ/main_test.go

```diff
@@ -91,6 +91,7 @@ func TestJVS(t *testing.T) {
 		ProjectID:          os.Getenv("PROJECT_ID"),
 		KeyName:            keyName,
 		Issuer:             "ci-test",
+		PluginAbsDir:       "/var/jvs/plugins",
```

Comment:
@yolocs I am not sure why this change is needed to make the integration test pass https://github.com/abcxyz/jvs/actions/runs/5479801335/jobs/9982133916. `go test` runs fine locally even without this change

Also it seems config.JustificationConfig is not loaded properly below? Otherwise it will pick up the default value

https://github.com/abcxyz/jvs/blob/1d813631a27efe248a1ea9675338d525dad93d78/test/integ/main_test.go#L90

Reply:
The short answer is: we don't load config in integ test. We directly construct a config in code (like the line you put).

The longer answer is: we should integ test more completely but didn't get time to. See https://github.com/abcxyz/jvs/issues/158

Reply:
I see, thanks for the context. Resolving this thread now